### PR TITLE
Add CONTRIBUTING.md to create a process of external contributors

### PR DESCRIPTION
**Pull request by legionth**
This PR adds `CONTRIBUTING.md` as recommend on: https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

I've added some rules here, but feel free to give me feedback on them 👍

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+When contributing to this service,
+please open an issue on
+[GitHub issues](https://github.com/onOfficeGmbH/sdk/issues)
+or send a mail to `apisupport@onoffice.de`
+this service before making a change.
```
Comment (jayay) [inline CONTRIBUTING.md]: "this service" can be removed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Bugs
+
+Bugs must be reported via the
+[GitHub issues](https://github.com/onOfficeGmbH/sdk/issues)
+Bug fixes and improvements can be provided via a Pull Request.
+
+A fix that redefines that behaviour of an implementation are considered
+as a [breaking change](#breaking-changes).
```
Comment (jayay) [inline CONTRIBUTING.md]: Maybe some like "A fix that introduces a change in behavior is considered a breaking change"?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Features
+
+Features and Breaking Changes must be discussed with the
+maintainers of this repository.
+
+Features that also changes the behaviour of existing implementations
```
Comment (jayay) [inline CONTRIBUTING.md]: "existing implementations" in terms of "current usages" or "existing setups"? I would rethink the terminology here (and above).

Features that also change~~s~~ the behaviour [...]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+please open an issue on
+[GitHub issues](https://github.com/onOfficeGmbH/sdk/issues)
+or send a mail to `apisupport@onoffice.de`
+before making a change.
+
+## Bugs
+
+Bugs must be reported via the
+[GitHub issues](https://github.com/onOfficeGmbH/sdk/issues)
+Bug fixes and improvements can be provided via a Pull Request.
```
Comment (dkaisers) [inline CONTRIBUTING.md]: I think you forgot to end the first sentence here after the link to the GitHub issues page. Maybe change the end of the sentence to something like "via the GitHub issues page". For me, there is currently just something missing for it to sound correct.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+A fix that introduces a change in behavior is considered a
+[breaking change](#breaking-changes).
+
+## Features
+
+Features and Breaking Changes must be discussed with the
+maintainers of this repository.
+
+Features that also change the behaviour of existing implementations
+are considered as [breaking changes](#breaking-changes).
```
Comment (dkaisers) [inline CONTRIBUTING.md]: I would probably remove the 'as'.

```suggestion
are considered [breaking changes](#breaking-changes).
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (legionth) [reply CONTRIBUTING.md]: IMO `current usages` and `existing setups` wouldn't be enough to clearify what is meant here.

"current usage" would indicate to much the users perspective, but implemenations can also be the internal behaviour. Like internal classes interact with each other or how class works internally.
The same goes for "existing setups". The description is to close to the enduser, while ignoring the internal environment.

But maybe that's just me 😄 

@dkaisers what do you think?

**Comment by legionth** [comment]
Changed `Contribution` chapter in README with reference to new file.

Happy Reviewing 👍

**Pull request merged**